tests/MQEstimator/test_mq.py

```diff
@@ -0,0 +1,168 @@
+from sage.all_cmdline import *
+import pytest
+from math import log2
+from cryptographic_estimators.MQEstimator import (
+    MQProblem,
+    MQEstimator,
+    MQProblem,
+    Bjorklund,
+    BooleanSolveFXL,
+    CGMTA,
+    Crossbred,
+    DinurFirst,
+    DinurSecond,
+    ExhaustiveSearch,
+    F5,
+    HybridF5,
+    KPG,
+    Lokshtanov,
+    MHT,
+)
+from tests.MQEstimator.legacy_implementations.mpkc.algorithms import (
+    Bjorklund as TestBjorklund,
+    BooleanSolveFXL as TestBooleanSolveFXL,
+    CGMTA as TestCGMTA,
+    Crossbred as TestCrossbred,
+    DinurFirst as TestDinurFirst,
+    DinurSecond as TestDinurSecond,
+    ExhaustiveSearch as TestExhaustiveSearch,
+    F5 as TestF5,
+    HybridF5 as TestHybridF5,
+    KPG as TestKPG,
+    Lokshtanov as TestLokshtanov,
+    MHT as TestMHT,
+)
+
+
+ranges = 0.01
+
+test_sets = [
+    # n   m   q
+    [50, 50, 2],
+    [70, 70, 4],
+    [50, 70, 8],
+    [120, 40, 8],
+]
+
+algos = [
+    Bjorklund,
```

Comment:
This is a little bit strange. Whenever you want to add a new MQAlgorithm, you need to this at 4 different places, only within this file. Is there a way to enumerate over the imports of an import statement, so this array is build up dynamically?